classes/install.yaml

```diff
@@ -11,12 +11,6 @@ inherit: [strip]
 # not pass --static to pkg-config.  Executables are still linked dynamically
 # because glibc cannot be linked statically).
 packageScript: |
-    # Install shared libraries only when cross compiling. On host builds only
-    # static libraries are installed.
-    if [[ "${AUTOCONF_BUILD:-unknown}" != "${AUTOCONF_HOST:-${AUTOCONF_BUILD:-unknown}}" ]]; then
-        INSTALL_SHARED=yes
-    fi
-
```

Comment:
This basically breaks the whole basement Linux build. For host builds no shared libraries must be used because they lead to tools that cannot be executed at all or not reliably.

Reply:
I'll check how to merge, that the user could overwrite the library type.